src/hw-monitor.cpp

```diff
@@ -136,36 +136,43 @@ namespace librealsense
             details.sendCommandData.data(),
             details.sizeOfSendCommandData);
 
-        send_hw_monitor_command(details);
+        if (locked_transfer)
+        {
+            return _locked_transfer->send_receive({ details.sendCommandData.begin(),details.sendCommandData.end()});
+        }
+        else
```

Comment:
Can remove the `else` clause and keep all the other code unindented

Reply:
Done.